src/v8runtime/V8Inspector.cpp

```diff
@@ -15,7 +15,11 @@
 #include "folly/json.h"
 #include "v8.h"
 
+#ifdef REACT_NATIVE_0_73
```

Comment:
```suggestion
#if REACT_NATIVE_TARGET_VERSION >= 73
```

we could reuse the `REACT_NATIVE_TARGET_VERSION`